.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+    "env": {
```

Comment:
`.eslintrc.json` files is duplicating the existing `.eslintrc` file - see https://github.com/strongloop-community/loopback-connector-cos/blob/master/.eslintrc

Please pick one only.

(I think you can safely remove `.eslintrc.json` from this pull request.)